www/success.html

```diff
@@ -10,19 +10,31 @@
       type="text/css"
       href="https://unpkg.com/@patternfly/patternfly@4.96.2/patternfly.min.css"
     />
+    <script lang="javascript">
+      function onLoad() {
+        window.location.href = 'podman-desktop:///preferences/authentication-providers'
```

Comment:
```suggestion
        window.location.href = 'podman-desktop://goto/preferences/authentication-providers'
```
would tell main side to call 

`this.apiSender.send('goto', '/preferences/authentication-providers')`

and on UI side App.svelte would have

```
window.events?.receive('goto', (location: unknown) => {
  const navRequest = location as string;
  router.goto(location);
});
```

Reply:
@dgolovin I think this can be discussed in follow-up

IMHO I think we need to ask to redirect to an extension page/command then it's the responsibility of the extension to do whatever the extension want. (There are some navigation API available for the extension)

But for now I think it's OK to just asking to bring back the main window of Podman Desktop on top by just asking `podman-desktop://`